microsoft/testsuites/xdp/xdpdump.py

```diff
@@ -89,6 +89,18 @@ def _install(self) -> bool:
         ).is_equal_to(self._code_path.parent)
         git.init_submodules(cwd=self._code_path)
 
+        # This workaround addresses a discrepancy in the bpf_perf_event_read_simple
```

Comment:
Does it work for all distros?

Reply:
I tested on mairner 2.0 and ubuntu 20.04 which don't change the compiler recently, I will cover more distro.

Reply:
I have tested the earliest images which pass this test case.